docs/configuration.md

```diff
@@ -160,6 +166,34 @@ info: CarbonAware.DataSources.Json.JsonDataSource[0]
     Reading Json data from /app/data-sources/json/mycustomfile.json
 ```
 
+### ElectricityMaps Configuration
+
+If using the ElectricityMaps data source, ElectricityMaps configuration is required.
+
+```json
+{
+    "APITokenHeader": "auth-token",
+    "APIToken": "",
+    "baseUrl": "https://api.electricitymap.org/v3/"
+}
+```
+
+> **Sign up for a free trial:** To get a free trial: https://api-portal.electricitymaps.com/catalog/prd_grncahbtg0kdh4kd
+
+#### API Token Header
+
+The API Token Header for ElectricityMaps is "auth-token".
+
+#### API Token
+
+The Electricity Maps token you receive with your account
+
+#### baseUrl
+
+The url to use when connecting to Electricity.  Defaults to ["https://api.electricitymap.org/v3/"]("https://api.electricitymap.org/v3/").
+
+In normal use, you shouldn't need to set this value, but this value can be used to enable integration testing scenarios or if the ElectricityMaps url should change in the future.
```

Comment:
Add note here that "normal use" is with a token, will need to be updated for free trial use